resources/views/blogs/first-feedback.blade.php

```diff
@@ -0,0 +1,38 @@
+<x-layout.main>
+    <body>
```

Comment:
`<body>` is a reserved tag which should only be used once in a specific part of an HTML document. Because it is already present in the layout, you should remove this here and all your other views (except layout.main.blade.php of course).